primitives/runtime/src/lib.rs

```diff
@@ -557,6 +557,8 @@ pub enum DispatchError {
 	Unavailable,
 	/// Root origin is not allowed.
 	RootNotAllowed,
+	/// The runtime is suspended and marks all transactions as failed.
```

Comment:
all transactions? even governance?

Reply:
Yea... kind of annoying. Governance is not mandatory (and it should not be), not sure if the `SafeMode` pallet could be enough.